test/index.js

```diff
@@ -213,3 +213,14 @@ it('should be able to decode a JPEG with options', function(t) {
   t.assert(rawImageData.data instanceof Uint8Array, 'data is a typed array');
   t.end();
 });
+
+it('should be able to decode a JPEG into RGB', function(t) {
+  var jpegData = fixture('grumpycat.jpg');
+  var rawImageData = jpeg.decode(new Uint8Array(jpegData), { useTArray: true, formatAsRGBA: false });
+  t.equal(rawImageData.width, 320);
+  t.equal(rawImageData.height, 180);
```

Comment:
```suggestion
  t.equal(rawImageData.height, 180);
  t.equal(rawImageData.data.length, 320 * 180 * 3);
```

just to be extra sure :)